pkg/features/kube_features.go

```diff
@@ -196,6 +196,11 @@ const (
 	// Requires careful consideration as it may lead to security issues.
 	// Deprecated: planned to be removed in 0.17
 	MultiKueueAllowInsecureKubeconfigs featuregate.Feature = "MultiKueueAllowInsecureKubeconfigs"
+
+	// owner: @pbundyra
+	//
+	// Enables reclaimable pods counting towards quota.
+	ReclaimablePods featuregate.Feature = "ReclaimablePods"
```

Comment:
Please add this feature on this table:

https://github.com/kubernetes-sigs/kueue/blob/1d717210e24bf980b48c765b7c1f6a605a1af943/site/content/en/docs/installation/_index.md?plain=1#L264-L301